nengo/builder/signal.py

```diff
@@ -238,9 +238,15 @@ def reshape(self, *shape):
         Any number of integers can be passed to this method,
         describing the desired shape of the returned signal.
         """
-        return Signal(self._initial_value.reshape(*shape),
+        if len(shape) == 1:
+            shape = shape[0]  # in case a tuple is passed in
```

Comment:
Why is it a problem if the shape is set to `(x,)` instead of `x`?

Reply:
It's not. The problem is because we're using the `*` for the arguments, if you call e.g. `reshape((1, 1))` you'll get `shape = ((1, 1),)` (wrapped inside an extra tuple), which is a problem.

Reply:
Ah, got ya.  We don't have to do this (it's not introduced by this PR), but I'd prefer just passing the shape as a tuple (i.e. `def reshape(self, shape)`), for consistency with `np.reshape`.  Not critical, but in case others felt the same way might as well do it now.

Reply:
`np.reshape` works either way, which is why we did it like this.

Reply:
Before I realized this simple trick would allow it to work both ways, I was considering dropping support for passing them separately. I'd be fine with that. This is a function that's mostly used internally, anyway, so dropping support would have little effect on user experience (though it's possible some backends are relying on it).

Reply:
I'm fine with leaving it in if other people find it useful 🤷‍♂️